code/modules/integrated_electronics/core/assemblies.dm

```diff
@@ -651,6 +654,26 @@
 	max_components = IC_MAX_SIZE_BASE / 2
 	max_complexity = IC_COMPLEXITY_BASE / 2
 
+/obj/item/electronic_assembly/small/implant
+	name = "electronic implant"
+	icon_state = "setup_implant"
+	desc = "It's a case, for building tiny-sized, implantable electronics with."
+	var/obj/item/implant/integrated_circuit/implant = null
```

Comment:
Is there any reason the properties of either an implant or electronic assemblies can't be componentized? This right here is extremely hacky

Reply:
Well... I was a bit too much of a lazy fuck here. I think it'll be better to componentize implants.